src/Core/Filesystem/Adapter/FlysystemAdapter.php

```diff
@@ -39,6 +39,7 @@ public function write($path, $content)
 
     public function delete($path)
     {
+        return;
```

Comment:
keep un-deleted. because when program run here, comodo is not verified for certain yet.